packages/relay/src/lib/eth.ts

```diff
@@ -493,7 +493,15 @@ export class EthImpl implements Eth {
           const accountCacheKey = `${constants.CACHE_KEY.ACCOUNT}_${transaction.to}`;
           let toAccount: object | null = this.cache.get(accountCacheKey);
           if (!toAccount) {
-            toAccount = await this.mirrorNodeClient.getAccount(transaction.to, requestId);
+            try {
+              toAccount = await this.mirrorNodeClient.getAccount(transaction.to, requestId);
+            } catch (error: any) {
+              if(error instanceof MirrorNodeClientError && error.statusCode == 404) {
+                toAccount = null;
+              } else {
+                throw error;
```

Comment:
Q: Should we log and throw `predefined.RESOURCE_NOT_FOUND` instead like in precheck?

Reply:
Not sure, since when is not found (404), we will return null instead.
So the only other option is an unexpected error or a (400 - Invalid Parameter [see docs](https://testnet.mirrornode.hedera.com/api/v1/docs/#/accounts/getAccountByIdOrAliasOrEvmAddress) )

I think that for Invalid Parameter we could also return Null or throw a related error, even thought right now we are returning null, I think we should throw an INVALID_PARAMETER error.

But for other errors, since we don't know what "unexpected error" would be, I think we should throw instead the same error we receive.

That said, I will log the error in both cases nevertheless.